src/libAtomVM/version.h

```diff
@@ -0,0 +1,23 @@
+/***************************************************************************
```

Comment:
I think this is generated at build time, do you have any good reason about why adding this file?
If you added this by mistake, please amend your previous commit.

Reply:
My bad! Will fix this ASAP